src/main/scala/com/gu/newsletterlistcleanse/services/ReportService.scala

```diff
@@ -17,7 +17,11 @@ class ReportService(amazonSns: AmazonSNS, config: NewsletterConfig) {
       .sortBy(_.newsletterName)
       .map { cl =>
         val newCount = cl.activeListLength - cl.deletionCandidates
-        val percentage = cl.deletionCandidates / cl.activeListLength
+        val percentage: Double = if (cl.activeListLength != 0) {
+          cl.deletionCandidates.toDouble / cl.activeListLength.toDouble * 100
```

Comment:
```suggestion
          (cl.deletionCandidates.toDouble / cl.activeListLength.toDouble) * 100
```
Make it clearer where we're multiplying by 100. Order of operation would cover it anyway but this reads more clearly.

Reply:
But it's the same right? `(1 / 5) * 100` is the mathematically the same as `1 / (5 * 100)`

Reply:
Not quite. `(1 / 5) * 100 = (1 * 100) / 5`. Otherwise you get `1/500 = 0.002`

Reply:
Yeah I just facepalmed